### ISSUE TITLE
Fix maap_base image in DIT

### ISSUE THREAD

**Issue by grallewellyn** [MAAP-Project/ZenHub#1143]
Based on [this build](https://repo.dit.maap-project.org/root/maap-workspaces/-/jobs/16124), it seems like 'mas.dit.maap-project.org/root/maap-workspaces/custom_images/maap_base:develop' is correct

This is registering the same algorithm (from the [tutorial](https://docs.maap-project.org/en/latest/technical_tutorials/dps_tutorial/dps_tutorial_demo.html#Register-the-Algorithm-with-DPS-using-the-Register-Algorithm-UI)) but with 
'mas.dit.maap-project.org/root/maap-workspaces/custom_images/maap_base:develop', the [build](https://repo.dit.maap-project.org/root/register-job/-/jobs/16371) fails 
'mas.dit.maap-project.org/root/maap-workspaces/base_images/python:develop', the [build](https://repo.dit.maap-project.org/root/register-job/-/jobs/16374) is successful 

This needs to be fixed because it is blocking: https://github.com/MAAP-Project/Community/issues/1056

**Comment by grallewellyn** [comment]
Building the same algorithm in ops using 'mas.dit.maap-project.org/root/maap-workspaces/custom_images/maap_base:develop', is successful https://repo.maap-project.org/root/register-job-hysds-v4/-/jobs/15285

@sujen1412 Do we want to be able to register algorithms using 'mas.dit.maap-project.org/root/maap-workspaces/custom_images/maap_base:develop' in DIT?

**Comment by sujen1412** [comment]
Yes, we want to use the same image. I will check what's going on.
